src/views/virtualmachinesinstances/utils/virtualMachinesInstancesRowFilter.ts

```diff
@@ -0,0 +1,51 @@
+import { V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { FilterValue, RowFilter } from '@openshift-console/dynamic-plugin-sdk';
+
+import { OS_TEMPLATE_LABEL } from './virtualMachinesInstancesConstans';
+import { osNames, vmiStatuses } from './virtualMachinesInstancesStatuses';
+
+const makeUserTitleOSName = (osName: string): string => {
+  //osName is in format: rhel8.5
+  const title = osName.split(/(\d.*)/, 2).join(' ');
+  //Will produce Rhel 8.5
```

Comment:
I don't know how the branding team will feel about us using incorrect capitalization of RHEL. Is this just a quick fix for now?

Reply:
I think you can just create an OS label object:

```export const OS_NAME_LABELS = {
  [OS_NAME_TYPES.rhel]: 'RHEL',
  [OS_NAME_TYPES.fedora]: 'Fedora',
  [OS_NAME_TYPES.centos]: 'CentOS',
  [OS_NAME_TYPES.windows]: 'Windows',
  [OS_NAME_TYPES.other]: 'Other',
...
};